app/utils.py

```diff
@@ -0,0 +1,34 @@
+from simple_salesforce import Salesforce
+from . import (
+    SF_USERNAME,
+    SF_PASSWORD,
+    SF_SECURITY_TOKEN,
+)
+
+
+def fetch_cases(comma_separated_case_numbers):
+    sf = Salesforce(username=SF_USERNAME, password=SF_PASSWORD, security_token=SF_SECURITY_TOKEN, domain='test')
+    case_numbers = comma_separated_case_numbers.split(',')
+    case_numbers_querystring = ','.join(repr(str(num)) for num in case_numbers) if case_numbers else "''"
+    cases = sf.query(
+        'SELECT id, Contract__c, Description, CaseNumber from Case WHERE CaseNumber IN ({})'
```

Comment:
Are you going to reuse this query in the future? Maybe you can define `Contract__c` `CaseNumber` etc, as constants or a class in your code so if it's something that change in Salesforce side, it will be easier to mantain.

Reply:
sounds great, I don't think this will change but it's more elegant to do so anyway.

Reply:
ok, so it's not important now, but keep in mind that idea if you see that `Salesforce` calls grows